data/crac-creation/crac-creator-cim/src/main/java/com/farao_community/farao/data/crac_creation/creator/cim/crac_creator/remedial_action/RemedialActionSeriesCreator.java

```diff
@@ -109,14 +161,14 @@ private void readAndAddRemedialAction(String cimSerieId, RemedialActionSeries re
             return;
         }
 
-        // --- Only import free-to-use remedial actions: there shouldn't be a shared domain tag
-        if (remedialActionSeries.getSharedDomain().size() > 0) {
+        // --- Shared domain
+        sharedDomain = null;
+        if (shouldReadSharedDomain && !remedialActionSeries.getSharedDomain().isEmpty()) {
             if (remedialActionSeries.getSharedDomain().size() > 1) {
-                remedialActionSeriesCreationContexts.add(RemedialActionSeriesCreationContext.notImported(createdRemedialActionId, ImportStatus.NOT_YET_HANDLED_BY_FARAO, String.format("Only free to use remedial actions series are handled. There are %s shared domains", remedialActionSeries.getSharedDomain().size())));
+                remedialActionSeriesCreationContexts.add(RemedialActionSeriesCreationContext.notImported(createdRemedialActionId, ImportStatus.NOT_YET_HANDLED_BY_FARAO, "RemedialActionSeries with multiple SharedDomain are not supported"));
```

Comment:
I was thinking that from a user perspective, it can make sense to define a remedial action that is available after a contingency in a given list of countries. This is why i used NOT_YET_HANDLED_BY_FARAO

Reply:
ok!